src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    public static final int ZERO_NUMBER = 0;
+    public static final int MAP_INDEX_KEY = 0;
+    public static final int MAP_INDEX_VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        Map<String, Integer> resultFileMap = new LinkedHashMap<>();
+        resultFileMap.put("supply", ZERO_NUMBER);
+        resultFileMap.put("buy", ZERO_NUMBER);
+        resultFileMap.put("result", ZERO_NUMBER);
+
+        try (BufferedReader readerFrom = new BufferedReader(new FileReader(fromFile));
+                BufferedWriter writeTo = new BufferedWriter(new FileWriter(toFile))) {
```

Comment:
please create separate private helper methods for reading and writing and move logic of reading and writing to them

you will call these methods inside getStatistic()